app/components/Texting/Texting.jsx

```diff
@@ -1,14 +1,16 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import ReactModal from 'react-modal';
-import { images } from 'assets';
+import configurations from '../../utils/whitelabel';
 import styles from './Texting.module.scss';
 import * as dataService from '../../utils/DataService';
 import FormView from './components/FormView';
 import Loader from '../ui/Loader';
 import SentView from './components/SentView';
 import ErrorView from './components/ErrorView';
 
+const { appImages: { icon } } = configurations;
```

Comment:
Why are you importing the `icon` function from the whitelabel instead of `assets`? None of the icons appear to be different for the different whitelabeled sites, so it doesn't need to be part of the whitelabel site. This comment applies to many of the other files in this PR diff, since a lot of them only import the `icon` function and nothing from the actual whitelabel configuration.